e2e/du_test.go

```diff
@@ -300,3 +300,21 @@ func TestDiskUsageByVersionIDAndAllVersions(t *testing.T) {
 		})
 	}
 }
+
+func TestDiskUsageEmptyBucket(t *testing.T) {
+	t.Parallel()
+
+	s3client, s5cmd := setup(t)
+
+	bucket := s3BucketFromTestName(t)
+	createBucket(t, s3client, bucket)
+
+	cmd := s5cmd("du", "s3://"+bucket)
+	result := icmd.RunCmd(cmd)
+
+	result.Assert(t, icmd.Success)
+
+	assertLines(t, result.Stdout(), map[int]compareFunc{
+		0: suffix(`0 bytes in 0 objects: s3://%v`, bucket),
+	})
+}
```

Comment:
This testcase is unrelated to this PR.

Reply:
[737](https://github.com/peak/s5cmd/pull/732) This PR contains changes from this other PR. I adjusted test cases according to the scenario where both PRs merged to the master branch. The other PR's changes must be present for the CI to pass in this branch. 
S, this PR can be merged after [737](https://github.com/peak/s5cmd/pull/732).

Reply:
https://github.com/peak/s5cmd/pull/732 merged. This PR just contains aimed changes to solve related issue now. It can be reviewed.